### DISCUSSION TITLE
proxy 能否在代理ajax 请求时注入请求头？

### DISCUSSION THREAD

**Discussion by Fengjing95** [umijs/umi#10807] [Q&A] [answered]
类似与 Nginx proxy_set_header 的效果

**Comment by fz6m** [comment, marked as answer]
[devServer 选项怎么配置](https://umijs.org/docs/introduce/faq#devserver-%E9%80%89%E9%A1%B9%E6%80%8E%E4%B9%88%E9%85%8D%E7%BD%AE)

**Reply by Fengjing95** [reply]
thanks
